src/types/error.ts

```diff
@@ -22,6 +22,14 @@ export enum OpenAttestationEthereumDocumentStoreRevokedCode {
   SKIPPED = 4,
   CONTRACT_NOT_FOUND = 404,
 }
+export enum OpenAttestationRevocationCode {
+  UNEXPECTED_ERROR = 0,
+  DOCUMENT_REVOKED = 1,
+  CONTRACT_ADDRESS_INVALID = 2,
+  ETHERS_UNHANDLED_ERROR = 3,
```

Comment:
What does this mean?

Reply:
iirc there are some ethers error correctly handled some not, this is in the case it's an unknown error from ethers